crates/sui-sdk/examples/utils/mod.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use reqwest::Client;
+use std::collections::HashMap;
+use sui_types::base_types::SuiAddress;
+// use sui_config::{sui_config_dir, SUI_CLIENT_CONFIG}; // uncomment if using a local wallet
```

Comment:
thoughts on just splitting this out into its own file?

Reply:
i was also curious about when we'd use WalletContext, so a separate example might help clarify that

Reply:
Makes sense, I will add a wallet example.